test/benchmarks/mapping_passes.py

```diff
@@ -35,7 +35,7 @@ def setup(self, n_qubits, depth):
             n_qubits, depth, measure=True, conditional=True, reset=True, seed=seed, max_operands=2
         )
         self.fresh_dag = circuit_to_dag(self.circuit)
-        self.basis_gates = ["u1", "u2", "u3", "cx", "iid"]
+        self.basis_gates = ["u1", "u2", "u3", "cx", "id"]
```

Comment:
This is clearly correct, but I think this might invalidate the results because it changes the hash of the code and asv will treat this as a new version. So I think we should fix it, but lets wait to merge this until after 1.3.0rc1 is tagged so we don't interfere with the collected data

Reply:
ping, 1.3.0rc1 is out.